webui/react/src/types.ts

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable-next-line @typescript-eslint/no-explicit-any */
+export type RawJson = Record<string, any>;
```

Comment:
**question:** probably a good reason for this, but just curious of as to using `any` instead of `unknown`

Reply:
if it's unknown we can't access any level in the object other than the first level. may if there was a way of defining a recursive Record<string, unknown>; type?

Reply:
I see, let's leave it